src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -370,4 +383,45 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
 
         onZoomChanged(zoom);
     };
+
+    private handleMapClick = (e: mapboxgl.EventData): void => {
+        const {
+            config: { viewport },
+            drillConfig,
+        } = this.props;
+        const { features, originalEvent, target } = e;
+        const { properties } = features[0];
+
+        // Disable drilling in edit/export mode
+        if (get(viewport, "frozen", false)) {
+            return;
+        }
+
+        // without this, the drill dialog will be closed
+        originalEvent.stopPropagation();
+
+        return handleGeoPushpinDrillEvent(drillConfig, this.getDrillProps(properties), target);
+    };
+
+    private handlePushpinMouseEnter = (e: mapboxgl.EventData): void => {
+        const { chart, props, tooltip } = this;
+        const { config } = props;
+        return handlePushpinMouseEnter(e, chart, tooltip, config);
+    };
+
+    private handlePushpinMouseLeave = (e: mapboxgl.EventData): void => {
+        const { chart, props, tooltip } = this;
+        const { config } = props;
+        return handlePushpinMouseLeave(e, chart, tooltip, config);
+    };
+
+    private getDrillProps(properties: GeoJSON.GeoJsonProperties): IDrillableProps {
+        let drillableProps: IDrillableProps;
+        try {
+            drillableProps = JSON.parse(properties.drillableProps);
+        } catch {
+            drillableProps = { drilldown: false };
```

Comment:
is "drilldown" flag somehow handled by geochart?

Reply:
No it is not. 
Removed. Thanks